### ISSUE TITLE
Allow files to be opened in web browser instead of editor from the terminal

### ISSUE THREAD

**Issue by rsokolewicz** [microsoft/vscode#231446]
In the integrated terminal, links to files can be clickable (ctrl + click) and the respective file will open in the editor. If a link points to a webpage, it will open that page in a web browser. I would like to have the option to choose to open a file in a web browser. 

## example

I'm writing a book using jupyter-book and when the build is done, the following is outputted in the terminal:

```
===============================================================================

Finished generating HTML for book.
Your book's HTML pages are here:
    book/_build/html/
You can look at your book by opening this file in a browser:
    book/_build/html/index.html
Or paste this line directly into your browser bar:
    file:///home/rsoko/book/book/_build/html/index.html            

===============================================================================

```

if I ctrl+click the last two links, they open in the editor. But I would prefer `file:///home/rsoko/private/IR_book/IR_book/_build/html/index.html` to open in my web browser instead. 

## desired outcome

Perhaps an option in `settings.json` with a regex pattern to override the default behavior of choosing between opening in browser or in editor. e.g. `"openFileInBrowser": ["*.html"]`

I'm also okay with interpreting this ticket as a bug, and claiming that all URLs that start with `file://` should be treated as links to be opened in the web browser, rather than in the editor.

**Comment by Timmmm** [comment]
> I'm also okay with interpreting this ticket as a bug, and claiming that all URLs that start with file:// should be treated as links to be opened in the web browser, rather than in the editor.

I agree, I think that should be the default behaviour and is also simplest to implement.

**Comment by Tyriar** [comment]
Related: https://github.com/microsoft/vscode/issues/216853

If we had a link-specific context menu as suggested in that issue we could have an open in browser option. That feels preferable to having some extension -> open behavior mapping setting as it doesn't require configuration.
